tpm2/constants.go

```diff
@@ -493,6 +493,7 @@ const (
 	CmdPolicyPCR                  tpmutil.Command = 0x0000017F
 	CmdReadClock                  tpmutil.Command = 0x00000181
 	CmdPCRExtend                  tpmutil.Command = 0x00000182
+	CmdPCRReset                   tpmutil.Command = 0x0000013D
```

Comment:
nit: this appears to be in sorted order by the CC value. (should go under CmdPCREvent)

Reply:
yes